src/PublicApiGenerator.Tool/PublicApiGenerator.Tool.csproj

```diff
@@ -1,14 +1,11 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net7</TargetFrameworks>
```

Comment:
Just wondering whether or not we could pick better targets here. `netcoreapp2.1` is relatively ancient, do we still need that? (I don't mind but would probably have picked `netcoreapp3.1` instead.) And instead of `net7` I'd probably choose `net6.0` which is an LTS release, and should work for .NET 7, too.

Reply:
I'm fine either way. I changed to 3.1 + 6.